Essentials/src/com/earth2me/essentials/ISettings.java

```diff
@@ -247,4 +249,12 @@
     boolean isSpawnOnJoin();
     
     boolean isTeleportToCenterLocation();
+    
+    boolean isCommandCooldownsEnabled();
+    
+    long getCommandCooldownMs(String label);
```

Comment:
I personally think this method name is confusing. Should be something more like `getRemainingCooldown`